exporting/exporting_engine.h

```diff
@@ -250,5 +251,7 @@ static inline void disable_instance(struct instance *instance)
 }
 
 #include "exporting/prometheus/prometheus.h"
+#include "exporting/prometheus/remote_write/remote_write.h"
+#include "exporting/aws_kinesis/aws_kinesis.h"
```

Comment:
We need these includes only in `exporting_engine.c` and with `#if HAVE_KINESIS` and `#if ENABLE_PROMETHEUS_REMOTE_WRITE`.